tensorflow/contrib/mpi_collectives/__init__.py

```diff
@@ -48,8 +48,10 @@
         print("MPI Size:", session.run(mpi.size()))
 ```
 
-@@rank
+@@init
 @@size
+@@rank
+@@local_rank
```

Comment:
Not necessarily something that needs to be changed, but more of a clarification.

These @@ s aren't doing anything, right? I think the tensorflow.org documentation excludes them because the documentation generation isn't configured with MPI (and the contrib inclusion is conditioned on that ./configure option). The other reason lots of contrib packages have them is interface sealing (i.e. delete all but these symbols), but there's no `remove_undocumented` (so `tf.contrib.mpi_collectives.tf` presumably exists along with a whole bunch of other symbols).

Reply:
I don't think they're doing anything. I added these to make sure the API was fully specified in the comment.